src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {` ${person.name}`}
```

Comment:
it's not good practice to set the spaces this way, instead of you need to use margins and padding
check other places